wg-data-protection/charter.md

```diff
@@ -0,0 +1,76 @@
+# WG Data Protection Charter
+
+This charter adheres to the [wg-governance] guidance as well as
+the general conventions described in the [Kubernetes Charter README] and
+the Roles and Organization Management outlined in [sig-governance], where
+applicable to a Working Group.
+
+
+## Scope
+
+Data protection in Kubernetes context typically involves backup and recovery
+of two types of entities:
+* Kubernetes object resources
+* Persistent volume data
+We consider it a complicated and layered problem, including backup and recovery
+at persistent volume level, application level, and cluster level. Part of the
+working group’s charter is to define what Kubernetes native constructs are
+required to achieve these goals.
+
+Data Protection in Kubernetes includes backup and restore of applications
+to a working state. Part of the working group’s charter is to define what
+this requires.
+
+The Data Protection Working Group is organized with the goal of providing
+a cross SIG forum to discuss how to support data protection in Kubernetes,
+identify missing functionality, and work together to design features that
+are needed to achieve the goal.
+
+### In scope
+
+* High level discussions on what it means to support data protection in Kubernetes at different level and how to do it.
+* Design discussions on specific topics related to data protection and disaster recovery support.
+* Document results of discussions and investigations in a linkable medium.
+
+Potential design topics include, but are not limited to the following:
+* Extract snapshot data without cloning
+* Volume backups
+* Data populator
+* Retrieve diffs between two snapshots (block and file level)
+* Consistency volume groups (group snapshot)
+* Application snapshot
+* Data protection policy
+* Data protection workflows
+
+### Out of Scope
+
+* Design discussions not related to data protection is out of scope.
```

Comment:
Can you be more specific? 

For example, what sorts of design discussions would belong better specifically within SIG Apps or SIG Storage independent of this WG?

Reply:
For example, how to enhancement volume placement and how to report volume health belong to SIG Storage and would not be a focus area of this WG.  Workload API designs for StatefulSet and Deployment belong to SIG Apps, however, this WG would be interested in figuring out how to backup and recover a StatefulSet or Deployment.

Reply:
thanks